src/gui/properties/peerlistwidget.cpp

```diff
@@ -102,7 +102,7 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
         hideColumn(PeerListDelegate::COUNTRY);
     // Ensure that at least one column is visible at all times
     bool atLeastOne = false;
-    for (unsigned int i = 0; i < PeerListDelegate::IP_HIDDEN; ++i) {
+    for (int i = 0; i < PeerListDelegate::IP_HIDDEN; ++i) {
```

Comment:
@thalieht:  Speaking of me not being a coder… 

Why did you change this (and the two similar cases below) from an unsigned type to a signed one?  

I cannot imagine a negative width measurement being valid—which makes `unsigned int` look rather useful to me.

Reply:
In this particular case there is no difference but there were other cases were `i` was converted implicitly to `int` anyway e.g. in line 115 and there was a warning. I just changed it here too for consistency.

Reply:
@thalieht:  I figured it might be a conversion issue.  

Thanks for the clarification.